app/views/invites/index.html.erb

```diff
@@ -0,0 +1,39 @@
+<div class="page-content invites">
+  <%= render 'layouts/header' %>
+  <%= render 'layouts/taskbar', origin: 'ROOT' %>
+  <div class="content pt-4">
+    <div class="container">
+      <h4>INVITE USERS</h4>
+
+
+      <div class="row invite-list-content">
+
+        <% @users.reject { |user| user == Event.find(params[:event_id]).creator }.each do |user| %>
+        <% invite = Invite.find_by(event_id: params[:event_id], user_id: user.id) %>
```

Comment:
same here, please use variables in your controllers that will hold logic like this, instead of putting it on the view. otherwise, you are not respecting the MVC pattern